ui/enhancement_shaman/inputs.ts

```diff
@@ -75,11 +75,15 @@ export const SyncTypeInput = InputHelpers.makeSpecOptionsEnumInput<Spec.SpecEnha
 export const EnhancementShamanRotationConfig = {
     inputs: 
         [
+        InputHelpers.makeRotationNumberInput<Spec.SpecEnhancementShaman>({
+            fieldName: "weaveLatency",
+            label: "Weaving Latency",
+            labelTooltip: "The amount of time to wait in milliseconds after an auto attack to begin casting a spell, only applies when Maelstrom Weapon has at least 1 stack.",
+        }),
         InputHelpers.makeRotationBooleanInput<Spec.SpecEnhancementShaman>({
             fieldName: 'lavaburstWeave',
             label: 'Enable Weaving Lava Burst',
             labelTooltip: 'Not particularily useful for dual wield, mostly a 2h option',
-            //enableWhen: (player: Player<Spec.SpecEnhancementShaman>) => player.getTalents().maelstromWeapon,
```

Comment:
this has been fixed in another PR that i just merged in, should be double checked

Reply:
Yeah ill catch it one of the merges :)